app/controllers/v0/virtual_agent/virtual_agent_appeal_controller.rb

```diff
@@ -154,6 +154,12 @@ def open_compensation?(appeal)
       def get_status_type_text(appeal_status, aoj)
         appeal_status_description = APPEAL_DESCRIPTIONS[appeal_status]
 
+        if appeal_status_description.nil?
+          appeal_status_description = 'Unknown Status'
+          unknown_status_error = StandardError.new("Unknown status: #{appeal_status} with AOJ: #{aoj}")
+          log_exception_to_sentry(unknown_status_error, { appeal_status => appeal_status, aoj => aoj })
```

Comment:
This all makes sense, but it's curious that this approach to logging exceptions to Sentry is different than the one in this PR https://github.com/department-of-veterans-affairs/vets-api/pull/12510/files I would think we would want a uniform way of signalling exceptional conditions and handle them in ways that, for example, make it clear _why_ it's happening -- especially in situations where we want different behavior in different environments.

Reply:
I think in this case we are creating an exception in an error state vs in the other we are just logging an exception that occurred. Let me know if that doesn't make sense.